### PR TITLE
Adding new company - CANCOM

### PR DESCRIPTION
**Pull request by Seenox**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [x] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: company-profiles/cancom.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+It has 4,000+ employees with offices across europe.
+
+## Remote status
+
+CANCOM is currently remote-friendly, and flexible.
+
+## Region
+
+Germany/Europe
```
Comment (dougaitken) [inline company-profiles/cancom.md:17]: The region here and in the readme file are for where the company hires. 

Is CANCOM only hiring in Germany or "across Europe" as mentioned in `Company size` ?

**Review by dougaitken — CHANGES_REQUESTED** [review]

File: company-profiles/cancom.md (hunk 1 above)
Comment (Seenox) [reply company-profiles/cancom.md:17]: @dougaitken thanks for your comment.
`Company size` part is where the offices are located and people are working from. It doesn't necessary mean that these offices/locations are _actively_ hiring. Or maybe I am missing something here.
Nevertheless, we are actively hiring in Germany, Austria and Slovakia. However, I am open to hiring people from anywhere in Europe, although not clearly stated on the website.

Do you want me to adjust Region in `company-profile` and `readme` to `Germany/Austria/Slovkia` for now?

Comment (dougaitken) [reply company-profiles/cancom.md:17]: Let's go for where you're actively hiring please @Seenox
Once this is merged, you can update whenever you'd like.

Comment (Seenox) [reply company-profiles/cancom.md]: @dougaitken Alright! I've pushed the changes. Please review (and adjust again if necessary by yourself) and merge.
Thanks!

**Comment by dougaitken** [comment]
Ah sorry, one more @Seenox - can you check the domain TLDs please? Readme has `.de` and the profile has `.com`

**Comment by Seenox** [comment]
I'm totally sorry, @dougaitken... It has ended up to be such an awkward (after-hours) merge request, after all! 😅 Should be fixed now.

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
All god @Seenox we got there! I'll get this merged in.

**Pull request merged**